bytes.go

```diff
@@ -77,30 +73,30 @@ func ToMegabytes(s string) (uint64, error) {
 // GB = G = GiB = 1024 * M
 // TB = T = TiB = 1024 * G
 func ToBytes(s string) (uint64, error) {
-	parts := bytesPattern.FindStringSubmatch(strings.TrimSpace(s))
-	if len(parts) < 3 {
-		return 0, invalidByteQuantityError
-	}
+	s = strings.TrimSpace(s)
+	s = strings.ToUpper(s)
+	for i, v := range s {
+		if unicode.IsLetter(v) {
```

Comment:
Can we use [IndexFunc](https://golang.org/pkg/strings/#IndexFunc) instead of the for loop ?

Reply:
hm.. ok.. but, what the reason?
```go
// IndexFunc returns the index into s of the first Unicode
// code point satisfying f(c), or -1 if none do.
func IndexFunc(s string, f func(rune) bool) int {
	return indexFunc(s, f, true)
}

// indexFunc is the same as IndexFunc except that if
// truth==false, the sense of the predicate function is
// inverted.
func indexFunc(s string, f func(rune) bool, truth bool) int {
	for i, r := range s {
		if f(r) == truth {
			return i
		}
	}
	return -1
}
```

Reply:
two additional calls